### PR TITLE
gg gold and track

### PR DESCRIPTION
**Pull request by evaldobratti**
new column in performance tab

I still need to change game log, to show which deaths were tracked

tell what you guys think about it

### CODE REVIEW

File: src/components/Match/matchColumns.jsx [hunk 1]
```diff
@@ -466,6 +466,31 @@ export const performanceColumns = [
...
+    tooltip: strings.tooltip_others,
+    field: 'performance_others',
+    sortFn: true,
+    displayFn: (row, col, field) => {
+      const comp = [];
+      if (field) {
+        if (field.tracked_deaths) {
+          comp.push(<div>
+            { inflictorWithValue('bounty_hunter_track', abbreviateNumber(field.tracked_deaths), '',
+              `${field.tracked_deaths} ${strings.tooltip_others_total_gold}
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: Maybe put these two elements in an array and do `.join('\n')` so it's more apparent

File: src/components/Match/matchColumns.jsx [hunk 2]
```diff
@@ -466,6 +466,31 @@ export const performanceColumns = [
...
+              ${field.track_gold} ${strings.tooltip_others_greevils_gold}`) }
+          </div>);
+        }
+        if (field.greevils_greed_gold) {
+          comp.push(<div>
+            { inflictorWithValue('alchemist_goblins_greed', abbreviateNumber(field.greevils_greed_gold), '',
+              `${field.greevils_greed_gold} ${strings.tooltip_others_greevils_gold}`) }
+          </div>);
+        }
+        return <div> { comp } </div>;
```
Comment (howardchung) [inline src/components/Match/matchColumns.jsx]: no need for the extra spaces in the div

File: src/lang/en-US.json [hunk 3]
```diff
@@ -702,6 +702,7 @@
   "th_permanent_buffs": "Buffs",
   "th_winner": "Winner",
   "th_played_with": "My Record With",
+  "th_others": "Others",
```
Comment (howardchung) [inline src/lang/en-US.json]: Just "Other" is better I think

File: src/lang/en-US.json [hunk 4]
```diff
@@ -841,6 +842,8 @@
   "tooltip_autoattack_other": "Auto Attack/Other",
   "tooltip_estimated_mmr": "MMR estimate based on the mean visible MMR of the recent matches played by this user",
   "tooltip_backpack": "Backpack",
+  "tooltip_others": "",
```
Comment (howardchung) [inline src/lang/en-US.json]: what is this empty string for?

File: src/lang/en-US.json [hunk 5]
```diff
@@ -862,5 +865,9 @@
   "vision_all_time": "All time",
   "vision_placed_observer": "placed Observer at",
   "vision_placed_sentry": "placed Sentry at",
-  "vision_ward_log": "Ward Log"
+  "vision_ward_log": "Ward Log",
+
+  "tooltip_others_tracked_deaths": "tracked deaths",
+  "tooltip_others_total_gold": "gold earnt from Track",
```
Comment (howardchung) [inline src/lang/en-US.json]: I think "earned" instead of "earnt"

File: src/components/Match/MatchLog.jsx [hunk 6]
```diff
@@ -96,7 +96,11 @@ const logColumns = [heroTdColumn, {
     switch (row.type) {
       case 'kills': {
         const hero = heroNames[row.detail] || {};
-        return <img src={`${API_HOST}${hero.img}`} className={styles.imgSmall} role="presentation" />;
+        const comp = [<img key="hero" src={`${API_HOST}${hero.img}`} className={styles.imgSmall} role="presentation" />];
+        if (row.tracked_death) {
+          comp.push('Tracked');
+        }
```
Comment (evaldobratti) [inline src/components/Match/MatchLog.jsx]: I don't know how to improve it, something like [this](http://imgur.com/kZLjsHb) maybe (that I have no idea how to do it), or remove it from log at all.

File: src/components/Match/MatchLog.jsx [hunk 7]
```diff
@@ -96,7 +96,11 @@ const logColumns = [heroTdColumn, {
     switch (row.type) {
       case 'kills': {
         const hero = heroNames[row.detail] || {};
-        return <img src={`${API_HOST}${hero.img}`} className={styles.imgSmall} role="presentation" />;
+        const comp = [<img key="hero" src={`${API_HOST}${hero.img}`} className={styles.imgSmall} role="presentation" />];
+        if (row.tracked_death) {
+          comp.push('Tracked');
```
Comment (howardchung) [inline src/components/Match/MatchLog.jsx]: move to string file?

File: src/components/Match/MatchLog.jsx [hunk 8]
```diff
@@ -96,7 +96,11 @@ const logColumns = [heroTdColumn, {
     switch (row.type) {
       case 'kills': {
         const hero = heroNames[row.detail] || {};
-        return <img src={`${API_HOST}${hero.img}`} className={styles.imgSmall} role="presentation" />;
+        const comp = [<img key="hero" src={`${API_HOST}${hero.img}`} className={styles.imgSmall} role="presentation" />];
```
Comment (howardchung) [inline src/components/Match/MatchLog.jsx]: Actually, I'mt not sure what you're trying to do here.  If you're trying to show some string if row.tracked_death is true, you should be able to draw another element in the div.  Having an array of an img element and a string seems kind of inconsistent

File: src/lang/en-US.json [hunk 9]
```diff
@@ -862,5 +864,9 @@
   "vision_all_time": "All time",
   "vision_placed_observer": "placed Observer at",
   "vision_placed_sentry": "placed Sentry at",
-  "vision_ward_log": "Ward Log"
+  "vision_ward_log": "Ward Log",
+
+  "tooltip_others_tracked_deaths": "tracked deaths",
```
Comment (howardchung) [inline src/lang/en-US.json]: move to the tooltip section with the other tooltip_ entries

**Comment by evaldobratti** [comment]
I'll try to think in something better to match log tracked kills.
I think this PR is done.

**Pull request merged**